### DISCUSSION TITLE
播放器定位 失效了吗？

### DISCUSSION THREAD

**Discussion by Henry-ZHR** [the1812/Bilibili-Evolved#4961] [综合讨论 / General]
b站现在改了宽屏，宽屏只会在看不到标题时触发定位，默认状态标题在最上面不触发

但即使关掉了默认播放器模式，播放器定位还是没有任何效果

**Comment by Geeyun-JY3** [comment]
<https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.1.6>：

> 以下功能已不再维护 (不再修复 bug 和添加新功能), 但是你仍然可以为其提供 Pull Request.
> 
> - 直播间自动抽奖
> - 默认播放器模式
> - 播放器置顶
> - 播放器定位

试试[`视频页默认定位`](https://github.com/the1812/Bilibili-Evolved/blob/77ada2305f3a4fba1c63dbe19cf08914cc787f38/doc/features/features.md#视频页默认定位)能不能满足你的要求。

**Reply by Henry-ZHR** [reply]
~~视频页默认定位也不行 位置测试是对的 但刷新又回到顶了 设置多少都是没有效果~~

新开标签页可以 刷新不生效 有点奇怪

**Comment by ghost** [comment]
其实像油管那样，播放器宽屏就不会有问题
